huxley/api/tests/test_committee_feedback.py

```diff
@@ -0,0 +1,123 @@
+# copyright (c) 2011-2015 Berkeley Model United Nations. All rights reserved.
+# Use of this source code is governed by a BSD License (see LICENSE).
+
+from django.core.exceptions import ValidationError
+
+from huxley.accounts.models import User
+from huxley.api import tests
+from huxley.api.tests import auto
+from huxley.utils.test import models
+import random
+
+
```

Comment:
You need to be more comprehensive in this file; you can't just test the parts you want to. Add tests for all users' behavior in all cases (think what would happen if we expect a chair not to be able to create feedback but someone accidentally adds code that does that; this test needs to catch it). Unit tests shouldn't test the current functionality they should test the contract with that part of the code; if the code is supposed to have a certain behavior that behavior needs to be tested even if at no point in the existing code is that behavior used.

Reply:
Yup I'll get to adding these! Shouldn't be too bad because we'll just be returning PermissionDenied on pretty much everything else.